Task12.cpp

```diff
@@ -0,0 +1,20 @@
+void getLongestPrefix(const char* word1, const char* word2, const char* word3, char* result)

```

Comment:
The function should have this type of declaration `void getLongestPrefix(const char* const* mtx, char* result)`. Try to implement the algorithm like so